src/core/audio/AudioAlsa.cpp

```diff
@@ -139,6 +142,118 @@ QString AudioAlsa::probeDevice()
 
 
 
+/**
+ * @brief Checks whether the ALSA device with the given name has the needed
+ *        capabilities for LMMS.
+ * @param deviceName Name of the device that is checked.
+ * @return If the device is usable for LMMS <tt>true</tt> is returned.
+ */
+bool hasCapabilities(char *device_name)
```

Comment:
Since this function is currently unused, I recommend commenting it out for the time being so that we don't get left with dead-code in our code-base if nobody gets around to integrating it.